content/kubernetes/reference/supported_k8s_distributions.md

```diff
@@ -44,94 +44,100 @@ Any distribution not listed below is not supported for production workloads.
 For details on this platform, see the Kubernetes [documentation](https://kubernetes.io/docs/home/supported-doc-versions/).
 
 {{<table-scrollable>}}
-|  | <nobr>**6.4.2-6**</nobr> | <nobr>**6.4.2-5**</nobr> | <nobr>**6.2.18-41**</nobr> | <nobr>**6.2.18-3**</nobr> | <nobr>**6.2.12-1**</nobr> | <nobr>**6.2.10-45**</nobr> | <nobr>**6.2.10-34**</nobr> | <nobr>**6.2.10-4**</nobr> | <nobr>**6.2.8-15**</nobr> | <nobr>**6.2.8-2**</nobr> | <nobr>**6.2.4-1**</nobr> |
```

Comment:
+1 for the new visual style for this page.
Did you also consider to inverse the versions order, both horizontal and vertical? i.e., have the operator versions increase left to right, and the k8s versions increase top to bottom? To my eyes this seems a bit more natural, not sure though. From a quick glance at other "compatibility matrices" this also seems like a more common way to lay it.

Reply:
We wanted the most recent to be the first thing the customer sees. Also if we did left to right, customers would have to scroll to see the most recent version.